pkg/query/top.go

```diff
@@ -73,10 +75,11 @@ func GenerateTopTable(ctx context.Context, p *parcaprofile.Profile) (*pb.Top, er
 	}
 
 	top := &pb.Top{
-		List:     list,
-		Reported: int32(len(list)),
-		Total:    int32(len(list)),
-		Unit:     p.Meta.SampleType.Unit,
+		List:       list,
+		Reported:   int32(len(list)),
+		Total:      int32(len(list)),
```

Comment:
I find this confusing. I think we should rather put the cumulative into the total value, and add a new field that's called "RowNumber" that more accurately reflects the meaning.